src/web/shared/component/Form.tsx

```diff
@@ -38,7 +39,7 @@ const Form = ({
   const [searchInput, setSearchInput] = useState('');
   const [selectedJuz, setSelectedJuz] = useState(undefined);
```

Comment:
Define the type when default is undefined
```suggestion
  const [selectedJuz, setSelectedJuz] = useState<SelectedJuzType>(undefined);
```

Reply:
it raise this error 
![image](https://github.com/user-attachments/assets/c28b711e-71e5-4175-a7f9-6f578bbb1e45)

Reply:
how about this?
```
const [selectedJuz, setSelectedJuz] = useState<Option>();
```

Reply:
refactored on other MR https://github.com/RR-Ardhika/murojaah/pull/37/commits/77932c06f32f6012a581054a78a03477d617797d